src/app/[locale]/(auth)/(routes)/signin/(components)/LoginDev/LoginDev.tsx

```diff
@@ -0,0 +1,17 @@
+// components
+import { GithubLoginButton } from '@/app/[locale]/(auth)/(components)/GithubLoginButton/GithubLoginButton'
+import Box from '@/components/Box/Box'
+import { useTranslations } from 'next-intl'
+
+const LoginDev = () => {
+  const t = useTranslations('loginDev')
```

Comment:
**nit:** We should have these translation namespaces delcared in some enum so it's for example: 

```const t = useTranslations(I18n.SignIn)```

Reply:
done